src/sentry/integrations/msteams/webhook.py

```diff
@@ -138,8 +145,8 @@ def post(self, request):
         if data["type"] == "message":
             # the only message events we care about are those which
             # are from a user submitting an option on a card, which
-            # will always contain an "actionType" in the data.
-            if not data["value"] or "actionType" not in data["value"]:
+            # will always contain an "payload.actionType" in the data.
+            if not data["value"].get("payload", {}).get("actionType"):
```

Comment:
If you're changing the latter two to `.get(...)`, why not the first one as well?